qa/tasks/cephfs/test_volumes.py

```diff
@@ -117,7 +122,53 @@ def test_connection_expiration(self):
         # Now wait for the mgr to expire the connection:
         self.wait_until_evicted(sessions[0]['id'], timeout=90)
 
+    def test_volume_create(self):
+        """
+        That the volume can be created and then cleans up
+        """
+        volname = self._generate_random_volume_name()
+        self._fs_cmd("volume", "create", volname)
```

Comment:
Have you tried testing this? I would expect a failure because multifs is not gated off with an experimental flag.

Reply:
Yes, Please see [1] and [2]. It's already there.

[1] https://github.com/ceph/ceph/blob/master/qa/tasks/cephfs/test_volumes.py#L45
[2] https://github.com/ceph/ceph/blob/master/qa/tasks/cephfs/test_volumes.py#L97